src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -7,15 +7,14 @@
 			>
 				<label
 					v-if="hideShowLendDropdown && !isLessThan25"
```

Comment:
Do we still need the `for` entry?

Reply:
I tried to have this dynamically similar to id, but unique id error appears again, having an static? is for sr only.

Reply:
Does removing `for` cause any issues?

Reply:
No issues, but not matching with every dynamic id... if I add a dynamic for like dynamic id to match them, score go down and unique id error appears again, do you agree having this static?  like `for="LoanAmountDropdown"` even they are not matching?

Reply:
Sometimes labels give extra functionality, like click to select the connected radio button. Are we using it for anything like that in this case?

Reply:
No visible extra functionality, this is just for screen readers... https://tailwindcss.com/docs/screen-readers
it was inherited from original component, so, we can remove it if not needed as visible I think, wdyt?

Reply:
How about remove the `label` and add `aria-label="Lend amount"` to the `kv-ui-select`

Reply:
Agreed!